pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -74,7 +75,13 @@ func totalImageSize(nodeInfo *schedulercache.NodeInfo, containers []v1.Container
 
 	imageSizes := nodeInfo.ImageSizes()
 	for _, container := range containers {
-		if size, ok := imageSizes[container.Image]; ok {
+		// append the default image tag if not specified
+		image := container.Image
+		if !strings.Contains(image, ":") {
+			image = image + ":latest"
```

Comment:
You need to add test to make sure every valid container image format (digest, tag, no tag) works with this logic.

Reply:
I see, will do - is there any other checking of the image name before the imagelocality sees it (my understanding is no, because the CRI has the final say on it anyway)?